src/brevitas_examples/imagenet_classification/ptq/ptq_evaluate.py

```diff
@@ -208,6 +249,7 @@ def main():
         f"{act_quant_calib_config}_"
         f"{args.weight_quant_calibration_type}_"
         f"{'bnc' if args.calibrate_bn else ''}")
+    # @TODO: include added options in configurations here
```

Comment:
Add f"{args.quant_format}

For the extra bitwidth parameters (ignore the layerwise first/last for now) add them like this:
f"{args.act_mantissa_bit_width if args.quant_format == 'float' else ''}"